openwisp_controller/config/migrations/0056_ow_zt_to_global.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations
+import json
+
+
+def change_owzt_to_global(apps, schema_editor):
+    Template = apps.get_model('config', 'Template')
+    for template in Template.objects.filter(
+        type='vpn', vpn__backend='openwisp_controller.vpn_backends.ZeroTier'):
+        config = json.loads(template.config)
+        if 'zerotier' in config:
+            for item in config.get('zerotier', []):
+                if item.get('name') == 'ow_zt':
+                    item['name'] = 'global'
```

Comment:
```suggestion
from django.db import migrations


def change_owzt_to_global(apps, schema_editor):
    Template = apps.get_model('config', 'Template')
    updated_templates = set()
    for template in Template.objects.filter(
        type='vpn', vpn__backend='openwisp_controller.vpn_backends.ZeroTier'
    ):
        if 'zerotier' in template.config:
            for item in template.config.get('zerotier', []):
                if item.get('name') == 'ow_zt':
                    item['name'] = 'global'
                    updated_templates.add(template)
    Template.objects.bulk_update(updated_templates, ['config'])
```

Reply:
@pandafy is this pr still in review?

Reply:
@filippociandy10 don't be afraid to ping us in [our dev chat](https://matrix.to/#/#openwisp_development:gitter.im), we currently have almost 150 open PRs across the entire project. Whenever you push changes, let us know.